tests/monotouch-test/SpriteKit/UniformTest.cs

```diff
@@ -0,0 +1,158 @@
+﻿#if !__WATCHOS__
+
+using System;
+#if XAMCORE_2_0
+using Foundation;
+using SpriteKit;
+using UIKit;
+using ObjCRuntime;
+#else
+using MonoTouch.Foundation;
+using MonoTouch.SpriteKit;
+using MonoTouch.UIKit;
+using MonoTouch.ObjCRuntime;
+#endif
+using OpenTK;
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.SpriteKit
+{
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class UniformTest
+	{
+		[TestFixtureSetUp]
+		public void Setup ()
+		{
+			if (!UIDevice.CurrentDevice.CheckSystemVersion (10, 0))
```

Comment:
Use the Xcode check instead of the version check:

    TestRuntime.AssertXcodeVersion (8, 0);